hazelcast/src/main/java/com/hazelcast/jet/impl/connector/ReadFilesP.java

```diff
@@ -174,8 +175,9 @@ private MetaSupplier(
         @Nonnull
         @Override
         public Function<? super Address, ? extends ProcessorSupplier> get(@Nonnull List<Address> addresses) {
-            return address -> ProcessorSupplier.of(() -> new ReadFilesP<>(directory, glob, sharedFileSystem,
-                    ignoreFileNotFound, readFileFn));
+            return address -> ProcessorSupplier.of(SecuredFunctions.readFilesProcessorFn(
```

Comment:
when backported this might be a breaking change for an upgrade - old lambda will be not present in the code